app/controllers/concerns/blacklight/search_context.rb

```diff
@@ -38,7 +38,7 @@ def set_current_search_session
   end
 
   def find_search_session
-    if agent_is_crawler?
+    if agent_is_crawler? || skip_session_tracking?
```

Comment:
Should we go ahead and push `agent_is_crawler?` into the `skip_session_tracking?` method?

Reply:
Or, should this check get pushed up to `record_search_parameters` or `before_action :set_current_search_session, if: ...` around line 15?

Reply:
👍